platform/admin/core/admin-core-logviewer/src/main/webapp/js/levels.js

```diff
@@ -16,12 +16,12 @@
 // Correspond to classes in log-entry.less
 // Also used to populate level-selector drop down list
 const levels = {
-  ERROR: 'errorLevel',
-  WARN: 'warnLevel',
-  INFO: 'infoLevel',
-  DEBUG: 'debugLevel',
+  ALL: undefined,
```

Comment:
Did we change this because the order was important? If so, we should mention that in the comment.